src/lib/flag.ts

```diff
@@ -7,8 +7,14 @@ export const noRebuild = flags.boolean({
   default: false,
 });
 
+export const build = flags.boolean({
+  description: 'rebuild the wasm before deploying.',
+  default: false,
+});
```

Comment:
Think this one should also default to true. I believe the majority will want to build and deploy in one go, and I want to optimize for the majority.